src/index.js

```diff
@@ -271,6 +271,10 @@ exports.init = async (Implementation) => {
       loadCollections(configStore.configDir);
     }
 
+    if (configStore?.Implementation?.RequestUnflattener) {
```

Comment:
[CAUTION] This might change depending on the discussion related to this comment : 
https://github.com/ForestAdmin/forest-express-mongoose/pull/644#discussion_r678287756